srcpkgs/zfs/patches/0003-patch-statx-for-musl.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/tests/zfs-tests/cmd/statx.c b/tests/zfs-tests/cmd/statx.c
```

Comment:
Include the patch header as well: https://github.com/openzfs/zfs/commit/8e9fcdac6f7bbc4ec92e89b8cdce1dcc404f1999.patch